cmd/argo/commands/list.go

```diff
@@ -87,6 +88,7 @@ func NewListCommand() *cobra.Command {
 	command.Flags().BoolVar(&listArgs.completed, "completed", false, "Show completed workflows. Mutually exclusive with --running.")
 	command.Flags().BoolVar(&listArgs.running, "running", false, "Show running workflows. Mutually exclusive with --completed.")
 	command.Flags().BoolVar(&listArgs.resubmitted, "resubmitted", false, "Show resubmitted workflows")
+	command.Flags().BoolVar(&listArgs.pending, "pending", false, "Show pending workflows")
```

Comment:
Technically speaking, I believe this is mutually exclusive with the other two statuses as well, as a workflow can only have [one phase](https://github.com/argoproj/argo-workflows/blob/6a1c7ef8abb3e9f6b310f9b35de94115115cf62b/pkg/apis/workflow/v1alpha1/workflow_phase.go#L6).

So this would need some more modifications